docs/Symbols.md

```diff
@@ -0,0 +1,98 @@
+# Symbols
+
+## Symbols filter settings
+
+By default Outline (symbols) display `DOM elements`, `processing instruction` and `DTD element, entity, attribute list declarations`.
+
+The DOM attributes and text nodes are not displayed in the outline. Indeed to prevent from good performance, symbols must be limitated; displaying attributes and text nodes in the outline could generate a lot of symbols.

```

Comment:
Indeed to ensure good performance, symbols must be limited;

Reply:
Fixed thanks for your suggestion.